web-module/src/main/webapp/resources/celJS/images/gallerylist-mobile-fix.js

```diff
@@ -1,22 +1,41 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
 (function(window, undefined) {
 
-  var galleryHideDesc= function(event) {
-    var galleryLink = event.findElement('.celements_gallery_link');
+  let galleryHideDesc= function(event) {
+    let galleryLink = event.findElement('.celements_gallery_link');
     if (galleryLink) {
       galleryLink.down('.text').hide();
       galleryLink.down('.background').hide();
     }
   };
 
-  var galleryShowDesc = function(event) {
-    var galleryLink = this;
+  let galleryShowDesc = function(event) {
+    let galleryLink = this;
     if (galleryLink) {
       galleryLink.down('.text').show();
       galleryLink.down('.background').show();
     }
   };
 
-  var galleryMobileFix = function() {
+  let galleryMobileFix = function() {
```

Comment:
all `let` can be replaced with `const` since the identifiers shouldn't be reassigned.